acadopy/bindings.pyx

```diff
@@ -387,7 +394,25 @@ cdef class Expression:
 
         return component
 
+    def __setitem__(self, int idx, value):
+
+        cdef Expression rhs, lhs
+        cdef acado.Expression* _rhs
+        cdef acado.Expression* _lhs
+        cdef unsigned int _idx = idx
+
+        rhs = as_expression(value)
+
+        cdef acado.Expression exp = deref(self._thisptr)(_idx)
+        
+        # FIXME: get the assignment operator= to work with
+        exp = deref(rhs._thisptr) # assignment
+
+
 cdef class ExpressionType(Expression):
+    # Is it ok to let users rely on the Expression.__cinit__.
```

Comment:
I suppose instantiating objects as `Expression` base classes makes more sense than abstracting the base class from an existing class...